release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -125,6 +125,13 @@ installed on the cluster nodes. Previously, OVS ran in a container on each node,
 managed by a DaemonSet. Using the host OVS eliminates any possible downtime from
 upgrading the containerized version of OVS.
 
+[id="ocp-4-5-termination-grace-period-support"]
+==== The terminationGracePeriod Parameter is now properly supported
+
+The `terminationGracePeriodseconds` parameter now works with the CRI-O 
```

Comment:
I've filed #23399 to fix `terminationGracePeriodseconds` -> `terminationGracePeriodSeconds`